Examples/KlaviyoSwiftExamples/Shared/AppDelegate.swift

```diff
@@ -34,14 +34,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         window?.makeKeyAndVisible()
 
         // STEP2: Setup Klaviyo SDK with api key
-        Klaviyo.setupWithPublicAPIKey(apiKey: "magpcN")
+        KlaviyoSDK().initialize(with: "magpcN")
```

Comment:
Did you want to change this back to apiKey?

Reply:
I personally think it reads better as `KlaviyoSDK().initialize(with: "magpcN")` but it's not clear as to what initializing the SDK with.  Although, `KlaviyoSDK().initialize(apiKey: "magpcN")` doesn't read as well it's clear as to what the SDK requires to initialize.